tests/libcxx/expected_results.txt

```diff
@@ -519,10 +520,10 @@ std/language.support/support.limits/limits/numeric.limits.members/traps.pass.cpp
 
 
 # *** STL BUGS ***
-# STL bug: VSO-121977 "<locale>: the enum value of std::money_base is not correct[libcxx]"
+# GH-1112 "<locale>: the enum value of std::money_base is not correct[libcxx]"

```

Comment:
Oooohhh, thanks for the effort to map these to Github bugs! 😍  Very nice. (No change requested.)